### PR TITLE
Complete of avatar

### PR DESCRIPTION
**Pull request by KBhautik6**
### Square profile

Screenshot 2023-12-27 at 3 45 13 PM

### Circle profile

Screenshot 2023-12-27 at 3 45 06 PM

- [x] Checking the code is duplicate or not?
- [x] Is these code has bug 
- [ ] Is code is check by shubhum sir or ankit sir?
- [ ] Are you going to merge this or not?

### CODE REVIEW

**Review by shubham-spreadd — APPROVED** [review]

File: src/stories/Avatar.stories.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    <Avatar url="https://i.pravatar.cc/80" Size={4.4} variant="square" />
+  </AvatarContainer>
+);
+
+export const CircularProfileAvatar = () => (
+  <AvatarContainer>
+    <Avatar url="https://i.pravatar.cc/40" Size={2.4} variant="circle" />
+    <Avatar url="https://i.pravatar.cc/60" Size={3.4} variant="circle" />
+    <Avatar url="https://i.pravatar.cc/80" Size={4.4} variant="circle" />
+  </AvatarContainer>
```
Comment (shubham-spreadd) [inline src/stories/Avatar.stories.js]: props should be similar to antd

**Review by shubham-spreadd — CHANGES_REQUESTED** [review]

File: src/stories/Avatar.stories.js (hunk 1 above)
Comment (KBhautik6) [reply src/stories/Avatar.stories.js]: Okay

**Review by ankitadrixus08 — APPROVED** [review]

File: src/stories/Avatar.Styled.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+export const StyledAvatarContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 10px;
+  align-items: center;
+`;
+
+export const DisplayImage = styled.img`
```
Comment (tarakhant1297) [inline src/stories/Avatar.Styled.js]: @KBhautik6 add styled prefix

**Review by tarakhant1297 — CHANGES_REQUESTED** [review]

File: src/stories/Avatar.Styled.js (hunk 2 above)
Comment (KBhautik6) [reply src/stories/Avatar.Styled.js]: Done

**Pull request merged**